Short-Answer/Algorithms_Answers.md

```diff
@@ -19,24 +20,19 @@ b)  sum = 0 // 0(1)
           for l in range(k + 1, 10 + k): //0(n) * 0(n) = 0(^n)
             sum += 1 // 0(1)
 ```
-- 0(1) + 0(n^4) = 0(n^4) The nested for loops dominates
+- 0(1) + 0(n^4) = 0(n^4) The nested for loops dominates.
 
```

Comment:
Almost; the time complexity is actually O(n^3), because the last loop is constant.